user_docs/en/userGuide.md

```diff
@@ -2516,6 +2516,16 @@ This option is only available if reporting of spelling errors is enabled in NVDA
 This option allows the user to control if key presses generated by applications such as on-screen keyboards and speech recognition software should be processed by NVDA.
 This option is on by default, though certain users may wish to turn this off, such as those typing Vietnamese with the UniKey typing software as it will  cause incorrect character input.
 
+##### Multiple press timeout {#MultiPressTimeout}
+
+Some NVDA keyboard gestures perform different actions based upon how many times the same key is pressed in rapid succession.
+An example of this is the "Report current character of navigator object" command.
+This command reports the character if pressed once, a phonetic description of the character if pressed twice, and the numeric value of the character if pressed three times.
+This option configures the timeout after which an additional press of the same key will start a new gesture, rather than being taken as a subsequent press of the first one.
+For the example command, a shorter timeout will cause two presses to the report the current character twice, rather than the phonetic description.
```

Comment:
@seanbudd, wonder if one is clearer (and is OK in English or not); feel free to accept or decline and merge as is.

```suggestion
For the example command, a too short timeout will cause two presses to the report the current character twice, rather than the phonetic description.
```

Reply:
Fixing another small typo with it

```suggestion
For the example command, a too short timeout will cause two presses to report the current character twice, rather than the phonetic description.
```